cluster.tf

```diff
@@ -1,7 +1,8 @@
 resource "aws_eks_cluster" "this" {
-  name     = "${var.cluster_name}"
-  role_arn = "${aws_iam_role.cluster.arn}"
-  version  = "${var.cluster_version}"
+  enabled_cluster_log_types = "${var.cluster_enabled_log_types}"
+  name                      = "${var.cluster_name}"
```

Comment:
A little bit nit picky can you can keep the `name` parameter at the top? It's more conventional 😄

Reply:
No problem. I just put it in alphabetical order since that's what it seemed like, but yes, I'll change it.